vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -200,13 +200,12 @@ testKubeBinariesPresent() {
   1.19.6-hotfix.20210118
   1.19.7-hotfix.20210310
   1.19.9-hotfix.20210322
-  1.19.12
```

Comment:
weird merge but we never deployed 1.19.12 and I think we might need the space anyway (1.20.2 is similarly out of latest 2 patch)

Reply:
I guess if we just replace 1, it should be fine..